labelbox/schema/project.py

```diff
@@ -145,6 +145,28 @@ def labels(self, datasets=None, order_by=None):
         return PaginatedCollection(self.client, query_str, {id_param: self.uid},
                                    ["project", "labels"], Label)
 
+    def queued_data_rows(self):
+        """
+        This function returns all DataRows that have not yet been labeled in this Project. 
+        It is helpful for MAL if a user only wants to upload inferences to DataRows that are still in the queue.
+        This is updated after attaching a Dataset. This will be accurate even if the Project isn't set up yet.
+
+        Returns:
+            All DataRows that are currently in the queue as a PaginatedCollection
+
+        """
+        id_param = "projectId"
+        query_str = """query GetProjectQueuedDatarowsPyApi($%s: ID!){
```

Comment:
Is there a reason the API use `strf` style string formatting as opposed to format strings or `.format`

I feel like either one would be better.

Reply:
I think the reason that it uses strf is because you don't need to add extra `{}` to support the other approaches. I just kept this pattern for consistency.

Reply:
E.g. to represent `{ id: 1}` you have to write `"{{ id: {}}}".format(1)` or `print(f"{{ id: {1}}}")` which is kind of gross.